subprojects/common/logger/src/main/kotlin/com.avito.logger/Logger.kt

```diff
@@ -2,9 +2,7 @@ package com.avito.logger
 
 interface Logger {
     fun debug(msg: String)
-    // TODO: implement in clients and remove empty implementation
-    fun info(msg: String) {} // empty implementation is for backward compatibility
+    fun info(msg: String)
     fun warn(msg: String, error: Throwable? = null)
-    fun exception(msg: String, error: Throwable)
```

Comment:
Looks excessed.  Delete for equality logger and ciLogger and merging them in the future